tools/reviewer/job/sync/GithubWriter.java

```diff
@@ -58,126 +60,165 @@ public long createPullRequest(PullRequest pullRequest, String repoOwner, String
                     .setRequestData(
                         CreatePullRequestRequest.CreatePullRequestRequestData.newBuilder()
                             .setTitle(pullRequest.getTitle())
-                            .setHead(pullRequest.getHead().getRef())
-                            .setBase(pullRequest.getBase().getRef())
-                            .setBody(pullRequest.getBody())
+                            .setHead(pullRequest.getHeadBranchName())
+                            .setBase(pullRequest.getBaseBranchName())
+                            // TODO: Set more informative message for default description
+                            // An attempt to create new Pull Request without specifying `body`
+                            // causes an error. We set the default value to avoid the error.
+                            .setBody(
+                                pullRequest.getBody().isEmpty()
+                                    ? "Default description"
```

Comment:
How about we change `s/Default description/No description set`?

Reply:
Done.